reactTest.html

```diff
@@ -0,0 +1,356 @@
+<!DOCTYPE html>
```

Comment:
It's an old file we were using for testing stuff I still reference a lot lol, you should git rm this file and push again @MCChung303